src/onnx/parse_generic_op.cpp

```diff
@@ -29,6 +29,8 @@ struct parse_generic_op : op_parser<parse_generic_op>
                 {"Floor", "floor"},
                 {"Gather", "gather"},
                 {"GatherND", "gathernd"},
+                {"Gelu", "gelu"},
+                {"GatherND", "gathernd"},
```

Comment:
These should be removed. GatherND is a duplicate, and the gelu is wrong as we dont have a gelu operator.

Reply:
done